kotest-assertions/kotest-assertions-shared/src/jvmTest/kotlin/com/sksamuel/kotest/ErrorCollectorTests.kt

```diff
@@ -0,0 +1,21 @@
+package com.sksamuel.kotest
+
+import io.kotest.assertions.AssertionFailedError
+import io.kotest.assertions.assertSoftly
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.style.FreeSpec
+import io.kotest.matchers.should
+import io.kotest.matchers.shouldBe
+
+class ErrorCollectorTests : FreeSpec(
```

Comment:
ah right there's no module for jsTest..

Reply:
Not sure what you meant with this 🤪  Did you mean that we should add a `jsTest` to check that I didn't break the js/desktop error collector?